Residents/MacOS/Makefile

```diff
@@ -1,7 +1,7 @@
 # Small makefile to build the residents easily
 
 all: residents
-	
```

Comment:
Some versions of make require this empty line with just a Tab for such targets to work correctly. We had problems with this in the past.

Reply:
In order to fix this, I need to know if this applies only to the above mentioned file, or if this was just an example.

As mentioned further down below, after amending the lost "tab lines" I'll need to change the EditorConfig settings form Makefiles by either:

- Disable "trailing whitespace" checks for specific files, or
- Just disable the "trailing whitespaces" checks for all Makefiles.

Even if we disable these checks, we'd still have the Tab-indentation validation (which seems more important), as well as the UTF-8 validation, and other small checks.

If trailing spaces creep in the Makefiles is not a huge deal, it's just noise; but if these enforced fixes could break their functionality, that would be far worste.